dbtwiz/model/create.py

```diff
@@ -94,27 +95,24 @@ def select_name(context):
     if context.get("name"):
         name = context.get("name")
     else:
-        name = input_text(            
-            f"Name your new model (it will be prefixed by {ModelBasePath(context.get('layer')).get_prefix(context.get('domain'))})",
+        model_base_path = ModelBasePath(context.get('layer'))
+        existing_models = list_domain_models(model_base_path, context.get('domain'))
+        name = input_text(
+            f"Name your new model (it will be prefixed by {model_base_path.get_prefix(context.get('domain'))})",
             validate=lambda text: (
                 all(
                     [
                         name_validator()(text) is True,
```

Comment:
Gjør `is True` noe her, eller er det kun for lesbarhetens skyld?

Reply:
Trur den er nødvendig, fordi `name_validator` er ein lambda funksjon som enten returnerer ein bool eller tekst som forklarer kva som var feil. Begge er truey, men det er kun dersom verdien er True at sjekken er godkjent.